WooCommerce/Classes/ViewRelated/Blaze/BlazeEditAd/BlazeEditAdViewModel.swift

```diff
@@ -45,14 +46,7 @@ final class BlazeEditAdViewModel: ObservableObject {
     }
     private var descriptionEmptyError: String?
 
-    // AI generation
-    @Published private(set) var generatingByAI = false
-
     var isSaveButtonEnabled: Bool {
-        guard generatingByAI == false else {
-            return false
-        }
-
```

Comment:
non-blocking suggestion: after this generatingByAI check is removed, perhaps this can be simplified as

```
var isSaveButtonEnabled: Bool {
    return editedAdData != nil && editedAdData != adData
}
```

Reply:
Thanks for the suggestion. But, I prefer the existing style of using `guard` and early exit to improve readability.